### PR TITLE
docs: categorize blog posts

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: common/type.ts [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+  | "Industry";
 
 export function postTagStyle(tag: PostTag): string {
   switch (tag) {
     case "Announcement":
       return "bg-indigo-100 text-indigo-800";
-    case "Education":
+    case "Engineering":
+      return "bg-gray-100 text-gray-800";
+    case "Explanation":
```
Comment (milasuperstar) [inline common/type.ts:14]: out of curiosity, why "explanation"? and not the original "education" (which digitalocean [also uses](https://www.digitalocean.com/blog))

File: content/blog/how-a-clickhouse-data-team-uses-bytebase-for-managing-schema-change.md [hunk 2]
```diff
@@ -3,12 +3,11 @@ title: How a Data Team Uses Bytebase to Manage Schema Change for ClickHouse
 author: Ningjing
 published_at: 2022/09/25 09:42:58
 feature_image: /static/blog/how-a-data-team-uses-bytebase-to-manage-schema-change-for-clickhouse/bytebase-for-feature-development.webp
-tags: Education
+tags: How-To
```
Comment (milasuperstar) [inline content/blog/how-a-clickhouse-data-team-uses-bytebase-for-managing-schema-change.md:6]: this is more like a use case IMO, not a how-to, maybe categorize it under industry? 🤔

File: content/blog/get-database-change-notification-via-webhook.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Get Database Change Notification via Webhook
 author: Candy
 published_at: 2022/05/16 09:42:58
 feature_image: /static/blog/get-database-change-notification-via-webhook/fish-hook.webp
-tags: Education
+tags: Explanation
```
[the remark this answers is not in the document]
Comment (adela-bytebase) [reply content/blog/get-database-change-notification-via-webhook.md:6]: Candy's previous product articles include "Announcement" and "How-to", which category should they belong to?

File: content/blog/gh-ost-based-online-schema-migration-for-mysql.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: gh-ost Based Online Schema Migration for MySQL
 author: Candy
 published_at: 2022/08/09 16:37:17
 feature_image: /static/blog/gh-ost-based-online-schema-migration-for-mysql/cover.webp
-tags: Education
+tags: Announcement
```
Comment (candy2255) [inline content/blog/gh-ost-based-online-schema-migration-for-mysql.md:6]: I think it is a "How-To"

File: content/blog/get-database-change-notification-via-webhook.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (candy2255) [reply content/blog/get-database-change-notification-via-webhook.md:6]: IMO, they are "How-To"

File: content/blog/gh-ost-based-online-schema-migration-for-mysql.md (hunk 4 above)
Comment (tianzhou) [reply content/blog/gh-ost-based-online-schema-migration-for-mysql.md:6]: The description is "Introduce gh-ost based online schema migration and how to perform an online schema migration without downtime in Bytebase." so I used Announcement

File: common/type.ts (hunk 1 above)
Comment (tianzhou) [reply common/type.ts:14]: Right

File: content/blog/get-database-change-notification-via-webhook.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tianzhou) [reply content/blog/get-database-change-notification-via-webhook.md:6]: Done

File: content/blog/how-a-clickhouse-data-team-uses-bytebase-for-managing-schema-change.md (hunk 2 above)
Comment (tianzhou) [reply content/blog/how-a-clickhouse-data-team-uses-bytebase-for-managing-schema-change.md:6]: Changed to case study

**Comment by tianzhou** [comment]
PTAL

**Review by milasuperstar — APPROVED** [review]

**Review by boojack — APPROVED** [review]
LGTM

**Review by adela-bytebase — APPROVED** [review]

**Pull request merged**